File: GameLibrary/src/cc/lib/logger/Logger.java
package cc.lib.logger;

/**
 * Created by chriscaron on 2/27/18.
 */

public interface Logger {
    void debug(String msg, Object... args);
    void info(String msg, Object... args);
    void error(String msg, Object... args);
    void error(Throwable e);
    void warn(String msg, Object... args);
    void verbose(String msg, Object ... args);
}

File: GameLibrary2/src/cc/lib/logger/Logger.kt
package cc.lib.logger

/**
 * Created by chriscaron on 2/27/18.
 */
interface Logger {
	fun debug(msg: String, vararg args: Any)
	fun info(msg: String, vararg args: Any)
	fun error(msg: String, vararg args: Any)
	fun error(e: Throwable)
	fun warn(msg: String, vararg args: Any)
	fun verbose(msg: String, vararg args: Any)
}